openid-connect-common/src/main/java/org/mitre/oauth2/model/AuthenticationHolderEntity.java

```diff
@@ -321,13 +321,20 @@ public Map<String, String> getRequestParameters() {
 	 * @param requestParameters the requestParameters to set
 	 */
 	public void setRequestParameters(Map<String, String> requestParameters) {
-		if (requestParameters == null) {
-			this.requestParameters = new HashMap<>();
-			return;
+		this.requestParameters = new HashMap<>();
+		if (requestParameters != null) {
+			for (Map.Entry<String, String> entry : requestParameters.entrySet()) {
+				String value = entry.getValue();
+				if (value != null && value.length() <= 2048) {
+					this.requestParameters.put(entry.getKey(), value);
+				} else if (value == null) {
+					logger.warn("The request parameter " + entry.getKey() + " has a null value.");
```

Comment:
Sonar would suggest this:
`logger.warn("The request parameter {} has a null value", entry.getKey())`
:slightly_smiling_face: